applications/crossbar/src/modules/cb_ledgers.erl

```diff
@@ -302,14 +302,18 @@ maybe_convert_units(_, Value) -> Value.
 process_action(Context, ?CREDIT, Ledger) ->
     case kz_ledger:credit(Ledger) of
         {'error', Reason} ->
-            crossbar_util:response('error', Reason, Context);
+            cb_context:setters(Context, [{fun cb_context:set_resp_status/2, 'error'}
```

Comment:
The standard way is to use `cb_context:add_system_error/2,3,4` or in the case of `kz_datamgr` use `crossbar_doc:handle_datamgr_errors/2,3`.